### PR TITLE
[Merged by Bors] - Static testnet configs

### PR DESCRIPTION
**Pull request by danielschonfeld**
## Issue Addressed

#1431 

## Proposed Changes

Added an archived zip file with required files manually

## Additional Info

1) Used zip, instead of tar.gz to add a single dependency instead of two.
2) I left the download from github code for now, waiting to hear if you'd like it cleaned up or left to be used for some tooling needs.

### CODE REVIEW

**Review by AgeManning — CHANGES_REQUESTED** [review]
Some minor comments

File: common/eth2_testnet_config/build.rs [hunk 1]
```diff
@@ -1,37 +1,53 @@
...
-                    panic!(e);
+        if !testnet_dir.exists() && archive_fullpath.exists() {
+            //uncompress archive and continue
+            let archive_file = File::open(&archive_fullpath).unwrap();
+            uncompress(archive_file);
+        }
+    }
+}
+
+fn uncompress(archive_file: File) {
```
Comment (AgeManning) [inline common/eth2_testnet_config/build.rs:29]: A general rule of thumb is to not use `unwrap()` in any working code in lighthouse (except for tests). We should always handle the errors and only panic as an absolute last resort. If we do have to panic, we should give a helpful description as to why. 

For this function, you can handle the errors more ergonomically by letting it return a result such as `Result<(), io::Error>`. Depending on the error types being produces, maybe you want to return a String here and print it in an explicit panic if it fails. 
The errors in the function can then either be mapped to a String via `map_err()` or potentially already use `io::Error`. Then you can replace the unwraps with `?`.

**Review by AgeManning — CHANGES_REQUESTED** [review]
Sorry my other comment didn't get included for some reason

**Comment by AgeManning** [comment]
Also run `cargo clippy` over this to fix up some of the formatting and errors to help pass the CI tests

**Comment by danielschonfeld** [comment]
> Also run `cargo clippy` over this to fix up some of the formatting and errors to help pass the CI tests

I actually did both fmt and clippy not sure why it produced that.  Same with what's mentioned above about a new line in Cargo.toml after eth2_config.

Do we want to get rid of all the old download from github code?

**Review by AgeManning — APPROVED** [review]
This looks good to me. 
I'll do a few more tests before merge and get some extra eyes on it.

Thanks

**Pull request closed without merging**